ui/app/routes/jobs/index.js

```diff
@@ -19,13 +18,15 @@ export default class IndexRoute extends Route.extend(
     },
   };
 
-  model(params) {
-    return RSVP.hash({
-      jobs: this.store
-        .query('job', { namespace: params.qpNamespace })
-        .catch(notifyForbidden(this)),
-      namespaces: this.store.findAll('namespace'),
-    });
+  async model(params) {
+    try {
+      return {
+        jobs: await this.store.query('job', { namespace: params.qpNamespace }),
+        namespaces: await this.store.findAll('namespace'),
+      };
+    } catch (e) {
+      notifyForbidden(this)(e);
+    }
```

Comment:
This migration definitely works in the sense that it returns the same data. However, the difference here between `RSVP.hash` or `Promise.all` is that by adding `await`, we reduce the parallelism here because the `await` will resolve sequentially, rather than running both requests at the same time. 

For example, the request tree _before_ this change looked like:

```
model
  ---> jobs
  ---> namespace
```

The data flow after the change using `await` in both keys now looks like this:

```
model
  ---> jobs
           ---> namespaces
```

In other words, users have to wait a little longer for stuff because they have to wait for jobs to resolve before getting namespaces, rather than fetching in parallel.

Rewriting the code to use `Promise.all` isn't as syntactically convenient, but it does allow us to make both requests at the same time and preserve the parallelism we got with `RSVP.hash`. What do you think about this suggestion?

```suggestion
  async model(params) {
    try {
      const [jobs, namespace] = await Promise.all([
        this.store.query('job', { namespace: params.qpNamespace }),
        this.store.findAll('namespace')
       ]);
       
     return { jobs, namespaces };
    } catch (e) {
      notifyForbidden(this, e);
    }
```

Reply:
OK, this is great feedback made the update!